assets/src/edit-story/constants.js

```diff
@@ -28,10 +28,13 @@ export const PAGE_NAV_PADDING = 60;
 export const PAGE_NAV_BUTTON_WIDTH = 40;
 export const PAGE_NAV_WIDTH = PAGE_NAV_PADDING + PAGE_NAV_BUTTON_WIDTH;
 
+export const PAGE_RATIO = 3 / 2;
 export const PAGE_WIDTH = 440;
 export const PAGE_HEIGHT = 660;
-export const DEFAULT_EDITOR_PAGE_WIDTH = PAGE_WIDTH;
-export const DEFAULT_EDITOR_PAGE_HEIGHT = PAGE_HEIGHT;
+
+export const ALLOWED_EDITOR_PAGE_WIDTHS = [440, 280, 240];
+
+export const PREVIEW_RATIO = 16 / 9;
```

Comment:
Ditto: `9 / 16`. I'd prefer we call it FULLBLEED_RATIO. IMHO, preview doesn't communicate the whole of this.